lib/utils/style-gather.js

```diff
@@ -7,18 +7,27 @@ The complete set of contributors may be found at http://polymer.github.io/CONTRI
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
+
+/**
+ * Module with utilities for collection CSS text from `<templates>`, external
+ * stylesheets, and `dom-module`s.
+ *
+ * @summary Module with utilities for collection CSS text from various sources.
+ */
+
+import { DomModule } from '../elements/dom-module.js';
 import { resolveCss } from './resolve-url.js';
 
 const MODULE_STYLE_LINK_SELECTOR = 'link[rel=import][type~=css]';
 const INCLUDE_ATTR = 'include';
 const SHADY_UNSCOPED_ATTR = 'shady-unscoped';
 
+/**
+ * @param {string} moduleId .
+ * @return {?DomModule} .
+ */
 function importModule(moduleId) {
-  const /** DomModule */ PolymerDomModule = customElements.get('dom-module');
-  if (!PolymerDomModule) {
-    return null;
-  }
-  return PolymerDomModule.import(moduleId);
+  return /** @type {?DomModule} */(DomModule.import(moduleId));
```

Comment:
Any recollection why the original code used `customElements.get` to get the DomModule class?  It was changed in this PR that doesn't have any detail: https://github.com/Polymer/polymer/pull/4748/files

Reply:
Yeah, Steve mentioned that ideally utils wouldn't depend on elements, and the `customElements.get` was there to avoid having that dependency, I guess in case someone wanted to use targeted imports to not load in dom-module.js.

I tried to preserve that by declaring some interfaces, but I'm worried it would be fragile, both to changes in DomModule, and in the face of aggressive compiler optimizations. And in practice essentially everyone who loads style-gather is going to load dom-module anyways...

Reply:
Seems ok for now.  We've recently discussed one way to make Polymer double-import safe would be to have <dom-module> only register if not already registered (first in wins), and then have _all_ `DomModule.import` call sites get `DomModule` from the CE registry like the old code here did.  But if we wanted to do that it would be a more sweeping change, and can do it separately.  Steve agreed this change should be fine for now.